src/ExpressReceiver.ts

```diff
@@ -124,7 +125,7 @@ export default class ExpressReceiver extends EventEmitter implements Receiver {
   }
 
   private errorHandler(err: any, _req: Request, _res: Response, next: NextFunction): void {
-    this.emit('error', err);
+    this.bolt?.handleError(err);
     // Forward to express' default error handler (which knows how to print stack traces in development)
     next(err);
   }
```

Comment:
Why doesn't this serve a purpose? It's still the global handler for express errors. 

I grant you that it's unlikely to be called, but it's not impossible.

Reply:
By the time express' error handler is triggered, the App's global error handler has already seen the error. The order is: listener, middleware, global error handler, receiver (which in some cases can go to express).

Invoking the global error handler after express sees the error would mean the global error handler has to deal with the same underlying error twice, at different points in the chain. I think that's confusing and unlikely that anyone will use it correctly.

If we remove it, express will generally do the right thing. In `NODE_ENV === 'development'` it will log the error, and if the response isn't already sent it will send a 500.